frontend/packages/pipelines-plugin/src/components/repository/__tests__/repository-utils.spec.tsx

```diff
@@ -56,4 +60,11 @@ describe('repository-util', () => {
       />,
     );
   });
+
+  it('sanitizeBranchName should return branch name', () => {
+    const branch1 = sanitizeBranchName('refs-heads-main');
```

Comment:
here is all our testcases in go : 

https://github.com/openshift-pipelines/pipelines-as-code/blame/a44ef08f70e12ea7d5bd5ef0b9ff816108038a07/pkg/formatting/vcs_test.go#L137

Reply:
Updated the test cases.